scripts/run_ansible_test

```diff
@@ -34,20 +34,21 @@ collection_path='/usr/share/ansible/collections/ansible_collections'
 case ${USE_VENV} in
     y|yes|true):
         ansible_test="${HOME}/test-python/bin/ansible-test"
-        collection_path="${HOME}.ansible/collections/ansible_collections"
+        collection_path="${HOME}/.ansible/collections/ansible_collections"
         ;;
 esac
 
-mkdir -p ${HOME}/code/ansible_collections/cifmw/general
-rsync -a --delete ${PROJECT_DIR}ci_framework/* ${HOME}/code/ansible_collections/cifmw/general/
-ln -fs ${collection_path}/* ${HOME}/code/ansible_collections/
+mkdir -p ${HOME}/code
+ansible-galaxy collection install git+file://${PROJECT_DIR}.git -p ${HOME}/code
```

Comment:
I think you don't want to keep the `.git` in there.

Reply:
thanks for the suggestion, testing it now